### ISSUE TITLE
No support for aria-selected for selected countries.

### ISSUE THREAD

**Issue by MikaelSvanasbakken** [iamstevendao/vue-tel-input#500]
The component currently lacks the aria-selected attribute to indicate the selected country in the dropdown - it would be nice to add to the component itself so that perhaps users can tab their way through the list.
